### PR TITLE
Add Splunk notice to README

### PR DESCRIPTION
**Pull request by theletterf**
This small PR adds the acquisition notice with a link to the README file.

### CODE REVIEW

**Review by owais — APPROVED** [review]

**Review by MrAlias — COMMENTED** [review]
I'm not sure this is necessary seeing as this project was created after the acquisition and all license headers attribute Splunk. That, and developers reading the README don't strike me as being interested in this kind of information. Is this is some policy we are trying across repositories?

Before we can merge this, please fix the markdown errors.

**Comment by theletterf** [comment]
Hi @MrAlias :

- &nbsp; if for formatting. GH renders the emoji with no separation from the word that comes after. Happy to apply a suggestion
- See the message in Slack as for what we're trying to accomplish.

Thanks!

File: README.md [hunk 1]
```diff
@@ -176,3 +176,5 @@ distribution of the [OpenTelemetry Go
 project](https://github.com/open-telemetry/opentelemetry-go). It is
 released under the terms of the Apache Software License version 2.0. See [the
 license file](./LICENSE) for more details.
+
+>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html) for more information.
```
Comment (MrAlias) [inline README.md:180]: To address the build failure:

> README.md:180:81 MD013/line-length Line length [Expected: 80; Actual: 183]

```suggestion
>ℹ️&nbsp;&nbsp;SignalFx was acquired by Splunk in October 2019. See [Splunk
SignalFx](https://www.splunk.com/en_us/investor-relations/acquisitions/signalfx.html)
for more information.
```

**Comment by theletterf** [comment]
@MrAlias What's the rationale for enforcing 80 columns per line? It's not enforced in most other repos, and I'd argue it's a bit outdated in most cases. Even Linus agrees :) https://linux.slashdot.org/story/20/05/31/211211/linus-torvalds-argues-against-80-column-line-length-coding-style-as-linux-kernel-deprecates-it

**Review by MrAlias — APPROVED** [review]

**Comment by MrAlias** [comment]
> @MrAlias What's the rationale for enforcing 80 columns per line? It's not enforced in most other repos, and I'd argue it's a bit outdated in most cases. Even Linus agrees :) https://linux.slashdot.org/story/20/05/31/211211/linus-torvalds-argues-against-80-column-line-length-coding-style-as-linux-kernel-deprecates-it

IIRC it's the default config of markdownlint that was setup when the project was created.

**Pull request merged**